src/main/java/org/jabref/gui/ai/components/summary/SummaryShowingComponent.java

```diff
@@ -32,20 +42,51 @@ public SummaryShowingComponent(Summary summary, Runnable regenerateCallback) {
 
     @FXML
     private void initialize() {
-        summaryTextArea.setText(summary.content());
+        initializeWebView();
+        updateContent(false); // Start in plain text mode
+        updateInfoText();
+    }
+
+    private void initializeWebView() {
+        contentWebView = WebViewStore.get();
+        VBox.setVgrow(contentWebView, Priority.ALWAYS);
+
+        themeManager.installCss(contentWebView.getEngine());
+
+        getChildren().addFirst(contentWebView);
+    }
 
+    private void updateContent(boolean isMarkdown) {
+        String content = summary.content();
+        if (isMarkdown) {
+            contentWebView.getEngine().loadContent(markdownFormatter.format(content));
+        } else {
+            contentWebView.getEngine().loadContent(
+                    "<body style='margin: 0; padding: 0; width: 100vw'>" +
```

Comment:
Could you add some padding? Maybe 5px, because rendered Markdown has padding, while plain one does not have